Server/Content/Sources/App/Models/Text.swift

```diff
@@ -0,0 +1,27 @@
+import Fluent
+import Vapor
+
+final class Text: Model, Content {
```

Comment:
Text is a great name but it is too broad I think, you even have a variable inside with the same name. Reading something like:
```swift
Text.text
```
is quite confusing though
Please try to come up with some more concrete name for this one